node/forest_libp2p/src/chain_exchange/behaviour.rs

```diff
@@ -64,7 +64,7 @@ impl ChainExchangeBehaviour {
         self.track_metrics();
         if let Some(tx) = self.response_channels.remove(request_id) {
             if let Err(err) = tx.send(Err(error.into())) {
-                warn!("{err}");
+                debug!("{err}");
```

Comment:
Could you write a comment here explaining how the errors are expected and not actually errors.

Reply:
Fixed